sources/packages/backend/libs/sims-db/src/entities/sfas-part-time-application.model.ts

```diff
@@ -56,4 +56,45 @@ export class SFASPartTimeApplications extends BaseSFASApplicationModel {
     nullable: false,
   })
   extractedAt: Date;
+  /**
+   * CSPT Award Amount (sail_extract_data.sail_cspt_amt).
+   */
+  @Column({
+    name: "cspt_award",
+    nullable: true,
+  })
+  CSPTAward: number;
+  /**
+   * CSGD Award Amount (sail_extract_data.sail_ptdep_amt).
+   */
+  @Column({
+    name: "csgd_award",
+    nullable: true,
+  })
+  CSGDAward: number;
+  /**
+   * BCAG-PT Amount (sail_extract_data.sail_bcag_amt).
+   */
+  @Column({
+    name: "bcag_award",
+    nullable: true,
+  })
+  BCAGAward: number;
+  /**
+   * CSLP Amount (sail_extract_data.sail_cslp_amt).
+   */
+  @Column({
+    name: "cslp_award",
+    nullable: true,
+  })
+  CSLPAward: number;
```

Comment:
Aren't the awards optional like the `programYearId`?

Reply:
Nice catch. Awards fields should not be optional, and I should make `programYearId` mandatory as well.

Reply:
@lewischen-aot As per the ticket programYearId is optional and there is no guarantee that it will be sent all the times. 

For other award fields, you can follow the same from what has been done for existing award columns unless we hear anything different from @JasonCTang on how awards are sent on bridge if student application is not eligible for it, (0 or NULL)